liferay-echo/liferay-portal
# LPS-192731 As an Account Administrator I want to be able to remove entries from the User invitation form

Motivation
=======
When an Account Admin invite a User to an Account, the Admin are able to create multiple entries inside the form but unable to remove any of them. The problem with this, that these entries consists a field that needs to be filled, and if the Admin click on the “Add entry” button they have to cancel the whole form, which leads to a bad user experience. The added entries needs to have an “x” button which removes the current entries.
https://liferay.atlassian.net/browse/LPS-189434

Proposed Solution
========
What we did to solve this problem is to add a remove button using ClayButtonWithIcon. This button is linked to an arrow function called onRemove which handles the removal of an item. Inside this function we use an openConfirmModal that asks the user if he is sure he wants to remove the icon, and if is affirmative, we remove the item from the array of items used to store them.

Steps to Verify
========
1. Go to Control Menu -> Accounts -> Create a new Account (+)
2. Open the new account created and click on (+) button -> Invite Members
3. Add different members and remove them by clicking on their removal button (x)
4. Check that the pop-up message appears and the removal action works correctly

<img width="1792" alt="Screenshot 2023-08-07 at 13 32 58" src="https://github.com/liferay-echo/liferay-portal/assets/91207948/2e62c5cd-f0aa-4b1d-a484-f576d016891e">
